_posts/2016-07-03-dart-new-tree-booster-provides-dropout.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "DART: new tree booster provides Dropout"
+date: 2016-07-03 00:00:00 +0900
+author: Yoshinori Nakano
+categories: xgboost
+comments: true
+---
+
+## Introduction
+Gradient Boosting Trees algorithms(especially [XGBoost](https://github.com/dmlc/xgboost)) give awesome performance in lots of [competitions](https://github.com/dmlc/xgboost/tree/master/demo#machine-learning-challenge-winning-solutions).
+They combine a huge number of regression trees with small learning rate.
+
+In this situation, trees added early are significance and trees added late are unimportant.
+DART algorithm drops trees added earlier to level contributions.
+
```

Comment:
Recently, Rasmi et.al proposed a new method to add dropout techniques from deep neural nets community to trees, and reported better results in some situations. We are glad to announce that DART is now supported in XGBoost, taking fully benefit of all xgboost optimizations to this new technique